sdk/search/Azure.Search.Documents/src/Generated/Models/SearchIndex.cs

```diff
@@ -30,10 +30,10 @@ public partial class SearchIndex
         /// <param name="encryptionKey"> A description of an encryption key that you create in Azure Key Vault. This key is used to provide an additional level of encryption-at-rest for your data when you want full assurance that no one, not even Microsoft, can decrypt your data in Azure Cognitive Search. Once you have encrypted your data, it will always remain encrypted. Azure Cognitive Search will ignore attempts to set this property to null. You can change this property as needed if you want to rotate your encryption key; Your data will be unaffected. Encryption with customer-managed keys is not available for free search services, and is only available for paid services created on or after January 1, 2019. </param>
         /// <param name="similarity"> The type of similarity algorithm to be used when scoring and ranking the documents matching a search query. The similarity algorithm can only be defined at index creation time and cannot be modified on existing indexes. If null, the ClassicSimilarity algorithm is used. </param>
         /// <param name="Etag"> The ETag of the index. </param>
-        internal SearchIndex(string name, IList<SearchField> fields, IList<ScoringProfile> scoringProfiles, string defaultScoringProfile, CorsOptions corsOptions, IList<SearchSuggester> suggesters, IList<LexicalAnalyzer> analyzers, IList<LexicalTokenizer> tokenizers, IList<TokenFilter> tokenFilters, IList<CharFilter> charFilters, SearchResourceEncryptionKey encryptionKey, SimilarityAlgorithm similarity, string Etag)
+        internal SearchIndex(string name, IList<SearchField> Fields, IList<ScoringProfile> scoringProfiles, string defaultScoringProfile, CorsOptions corsOptions, IList<SearchSuggester> suggesters, IList<LexicalAnalyzer> analyzers, IList<LexicalTokenizer> tokenizers, IList<TokenFilter> tokenFilters, IList<CharFilter> charFilters, SearchResourceEncryptionKey encryptionKey, SimilarityAlgorithm similarity, string Etag)
```

Comment:
@pakrym, it's not an issue, but I found this unusual with the generator. When I changed to using a field, the parameter name casing was changed to PascalCase. Still internal/private accessibility, so not a problem.

Reply:
It's a bug in our identifier generation logic. Can you file it, please?

Reply:
https://github.com/Azure/autorest.csharp/issues/925